.github/workflows/integration-examples.yml

```diff
@@ -0,0 +1,42 @@
+name: Integration examples
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  suapp-examples:
+    name: Devenv
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v3
+        with:
+          go-version: ^1.20
```

Comment:
1.21?

Reply:
I copied it from another workflow. Do not have any strong opinion about it.

Reply:
+1 to version 1.21 because of WASM/WASI support, which I imagine we'll eventually want.